### ISSUE TITLE
Error code 199-9999 when attempting to launch

### ISSUE THREAD

**Issue by CreeperMario** [shinyquagsire23/woominstaller#3]
Whenever I try to launch Woomïnstaller from the Wii U Menu, I receive error 199-9999. A popup screen shows up, but all it says is "An error has occurred. If this error persists, contact Nintendo repairs technician" etc.

In most cases reported on the GBATemp forums, error 199-9999 is related to signature mismatches. Though I do indeed have signature checks disabled, as Woomïnstaller is only present on my redNAND setup (using Mocha btw), and I was able to successfully open the Homebrew Launcher RPX from my menu, not to mention the installation of Woomïnstaller itself succeeded, all within the same boot cycle.

If you're interested, I installed Woomïnstaller on my console by downloading your Git repo, building it (100 or so warnings, probably related to me compiling it with devkitPPC r28 and all of its changed type declarations. 😜) and then copying the FST contents to my SD Card, which I installed on my console using WUPInstaller Y Mod.

The received error message that I mentioned above was very vague about the problem, though I may have found more valuable information in my system log. (Thanks to VinsCool for showing me this trick). I have attached an exert from said logs (/vol/system_slc/usr/logs/28.log I believe it was), which were extracted from my console by rebooting after the 199-9999 error, booting my redNAND and then launching FTPiiU Everywhere.

The log I have provided begins from the point I launched Mocha redNAND, walks through with me as I launch HBL, launch WUPInstaller, use it to install Woomïnstaller's FST data, close it and return to HBL, close it and return to the Wii U Menu and finally attempt to launch Woomïnstaller, which results in error 199-9999.

[syslog.zip](https://github.com/shinyquagsire23/woominstaller/files/673823/syslog.zip)

**Comment by shinyquagsire23** [comment]
An important thing to note is that SHA hashes are checked on meta/bootmovie.h264 and meta/bootLogoTex.tga. I don't include the files usually found in retail applications because they're likely copyrighted. It *does* seem that ACP is patched for that in Mocha though, so I'm not sure what else it could be... Is it an issue with makefst?

**Comment by CreeperMario** [comment]
> Is it an issue with makefst?

Probably not, I tried building it with NUSPacker and received the same error.

**Comment by Mrrraou** [comment]
installed the fst with ios-mcp from the ppc side, got no issues whatsoever for starting it or installing it. i used my own cfw though. ios-acp was indeed patched on my side, too.

**Comment by CreeperMario** [comment]
So I began looking into this further today.

Earlier this year, I was working on a program called Space Game. I had tried building it using a particular title ID, and had received similar errors trying to launch it as I did with WoomÏnstaller, WoomÏnstaller GX2 and Endless Nuclear Kittens.

I was given a tipoff by *some person who will not be named without his/her permission* who mentioned that he/she had more luck using title IDs that began with "1" e.g. "00050000**1**3374842" for HBL. I tried this in Space Game and it resolved all my issues.

So I tried this logic on WoomÏnstaller, downloading the source and re-building it, after changing the title ID in app.xml and meta.xml from "00050000**77**6D7900" to "00050000**13**6D7900". And surely enough, after installing the package, it began working flawlessly. Even across reboots.

And so I tried the same thing with WoomÏnstaller GX2, and had success.

I haven't yet tried Endless Nuclear Kittens, as I have yet to get minlibd to build. 😄

TL;DR - The title ID you chose is problematic - changing it to something like "000500001xxxxxxx" fixes all issues on my end.
